{{cookiecutter.project_slug}}/requirements/local.txt

```diff
@@ -1,4 +1,4 @@
--r base.txt
+-r production.txt
```

Comment:
Next step: merge `base.txt` and `production.txt`? 😄

Reply:
I've suggested that before in #2810 but felt out of scope for this PR.

Reply:
Oh yes, I agree: 100% out of scope. I was merely pointing out that it might help us get there without changing too much our requirements files.